exercises/array-map/main.js

```diff
@@ -0,0 +1,17 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(numbers => numbers * 2);
+
+const prices = numbers.map(numbers => `$${numbers.toFixed(2)}`);
+
+const upperCased = languages.map(languages => languages.toUpperCase());
+
+const firstLetters = languages.map(languages => languages[0]);
```

Comment:
The parameter of your arrow function is actually _one_ element of the array, so you should name it in the singular form like `number` or `language`.